### PR TITLE
[REF][8.0][BR000973][T22092]Refactor elico_project into several distinct modules[hr_timesheet_line_readonly]

### PR DESCRIPTION
**Pull request by dongshukai**
Submit the PR for module hr_timesheet_line_readonly

### CODE REVIEW

File: hr_timesheet_line_readonly/README.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * have basic modules installed (hr_timesheet)
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/Elico-Corp/odoo/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/Elico-Corp/odoo/issues/new?body=module:%20hr_timesheet_line_readonly%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (elicoidal) [inline hr_timesheet_line_readonly/README.rst]: Remove this line

File: hr_timesheet_line_readonly/README.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+To install this module, you need to:
+
+ * have basic modules installed (hr_timesheet)
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/Elico-Corp/odoo/issues>`_.
```
Comment (elicoidal) [inline hr_timesheet_line_readonly/README.rst]: URL should be https://github.com/Elico-Corp/odoo-addons/issues

**Review by elicoidal — CHANGES_REQUESTED** [review]
some details
LGTM
@victormartinelicocorp

**Comment by dongshukai** [comment]
@Elico-Corp/odoo-reviewers 
pls check

**Review by elicoidal — APPROVED** [review]

**Pull request merged**